README.md

```diff
@@ -41,6 +42,8 @@
 | Whitepaper               | <https://docs.okp4.network/whitepaper/abstract>  | [OKP4](https://github.com/okp4) |
 | Nodes & Validator Guide  | <https://docs.okp4.network/nodes/introduction>   | [OKP4](https://github.com/okp4) |
 | Nodes & Validator Guide  | <https://nodes.guru/okp4/setup-guide/en>         | [Nodes Guru](https://nodes.guru/) |
+| Nodes & Validator Guide  | <https://nodeist.net/t/okp4/Setupen/>         | [Nodes Guru](https://nodes.guru/) |
+| Nodes & Validator Guide  | <https://nodes.guru/okp4/setup-guide/en>         | [Nodeist](https://nodeist.net/) |
```

Comment:
It's seems that the url https://nodes.guru/okp4/setup-guide/en is written twice and the https://nodeist.net/t/okp4/Setupen/ url's is not in the good row. 

Here the correction if it's good : 
```suggestion
| Nodes & Validator Guide  | <https://nodes.guru/okp4/setup-guide/en>         | [Nodes Guru](https://nodes.guru/) |
| Nodes & Validator Guide  | <https://nodeist.net/t/okp4/Setupen/>         | [Nodeist](https://nodeist.net/) |
```